ynput/ayon-core
# Bugfix/houdini redshift render aovs in one layer

## Changelog Description
It was reported that Redshift jobs don't render all aov paths in one exr layer even when setting render mode to `Full Multi-Layered EXR File` or when enabling `MultipartExr`
![image](https://github.com/ynput/ayon-core/assets/20871534/61611057-22a7-4d81-a39a-0cd4a987db4b)



## Testing notes:
1. Create a redshift instance in `Full Multi-Layered EXR File` from creator
2. Publish to farm
3. Check if it outputs one exr sequence and it contains all the paths
